Documentation/ProjectFile/prj/processes/process/THERMAL_TWOPHASE_FLOW_PP/material_property/t_specific_heat_capacity_vapor.md

```diff
@@ -0,0 +1 @@
+Specific heat capacity of the water vapor.
```

Comment:
In this implementation, wetting phase mainly represents liquid water. in that case water vapor always presents in the system.